client/src/pages/Programmes/ExistingProgramme.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Flex, Text, Box, IconButton } from "@chakra-ui/core";
+import OutlineButton from "../../components/OutlineButton";
```

Comment:
Similar to my comment above export the component from `index.js` in the components folder so we have 

`import {OutlineButton} from "../../components";`